main.tex

```diff
@@ -354,16 +358,18 @@ \section*{Introduction}
 Thus, it can be applied to many fields of science, art, and technology, including biochemical and cellular simulation models.
 By exploiting standardized provenance data models, such as PROV-DM, this information is presented in a structured, queryable, and graphical form~\cite{Ruscheinski2018, Ruscheinski2017}.
 In addition to providing crucial information about the generation of an individual simulation model and, thus, facilitating its reuse, provenance can be applied to reveal and exploit relationships between different simulation models.
+% TODO the above structure should be changed?
 
 % What is our goal of this publication? And what is the Wnt signaling pathway?
 In this publication, we will identify and structure relevant information needed for the provenance of simulation studies and elucidate how a family of simulation models can be established through relating the models to each other.
```

Comment:
"elucidate" is a new word to me! not sure if it should be changed to "explain"? If this is actually a common word in academia this disregard this comment